move chained proxies (not) (#2281)
uyuni-project/uyuni-docs: modules/client-configuration/pages/client-proxy.adoc

You can move Salt and Salt SSH push clients between proxies without the need to repeat the registration process.

[IMPORTANT]
====
You cannot move chained proxies.

> In such a case, create a new proxy, move the clients, and delete the old proxy.

Author: ```suggestion
Instead of moving a chained proxy, create a new proxy, move the clients, and delete the old proxy.
```
